sdk/communication/azure-communication-chat/src/main/java/com/azure/android/communication/chat/ChatThreadAsyncClient.java

```diff
@@ -173,36 +172,36 @@ public String getChatThreadId() {
     /**
      * Adds participants to a thread. If participants already exist, no change occurs.
      *
-     * @param options Options for adding participants.
+     * @param participants Collection of participants to add.
```

Comment:
Since there's a Java `Collection<T>` class, should we rephrase this to something like the following?

```suggestion
     * @param participants Participants to add.
```